app/request.go

```diff
@@ -66,8 +91,102 @@ func NewRequestCommand(getRequestClientFn GetRequestClientFn) (CommandOut, error
 				},
 			},
 			Action: func(ctx *cli.Context) error {
-				return c.getRequestStatus(ctx.String("request-id"))
+				res, err := c.getRequestStatus(ctx, ctx.String("request-id"))
+				if err != nil {
+					return err
+				}
+				return PrintProto(res)
+			},
+		}, {
+			Name:    "wait",
+			Usage:   "wait till the request completes",
```

Comment:
[Nit] "wait for the request to complete"

Reply:
fixed